BachUZ/Program.cs

```diff
@@ -18,14 +18,14 @@ public static void Main(string[] args)
 
         public async Task MainAsync()
         {
+            await UZScheduleService.PopulateScheduleCache();
```

Comment:
I think we shouldn't ask for schedule each time we start, perhaps it should happen only when user asks for it